lib/tasks/pubmed.rake

```diff
@@ -10,4 +10,36 @@ namespace :pubmed do
     doc = client.fetch_records_for_pmid_list(pmids)
     puts doc # XML document
   end
+
```

Comment:
will describe in a comment -- it is just a plain text file with no header, one row per PMID (no other columns)